bindings/c/Makefile

```diff
@@ -0,0 +1,24 @@
+#
+# Build the example client
+#
+
+RPATH=$(PWD)/../../target/debug
+CFLAGS = -I./include
+LDFLAGS = -L$(RPATH) -Wl,-rpath,$(RPATH)
+LIBS = -lopendal
+OBJ_DIR=obj
+
+all: $(OBJ_DIR) build test
+
+$(OBJ_DIR):
+	mkdir -p $@
+
+build:
+	cargo build
```

Comment:
Honestly, I don't want to work with `Makefile`...

Reply:
We have so many options available to us that Makefile is only temporary.